### ISSUE TITLE
Indent Fix for Multi-line Comments Only Fixes First Line

### ISSUE THREAD

**Issue by epmatsw** [eslint/eslint#8444]
**Tell us about your environment**

* **ESLint Version:** 4.0.0-alpha.0
* **Node Version:** 7.7.4
* **npm Version:** 4.5.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

```
{
    "extends": "eslint:recommended",
    "rules": {
        "indent": 2,
    }
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
       /*
        * test1
        */

/*
 * test2
 */

{
   /*
    * test 3
    */
}

```
Note the 7 leading spaces in line 1 of test1 and 3 leading spaces in the first comment line of test3.

**What did you expect to happen?**
I expected the 2nd and 3rd lines of test1 to align with the first line. I also expected the alignment of test2 and test3 to be consistent (aligned asterisks, or aligned first characters).

**What actually happened? Please include the actual, raw output from ESLint.**
```js
/*
        * test1
        */

/*
 * test2
 */

{
    /*
    * test 3
    */
}

```

**Comment by not-an-aardvark** [comment]
Thanks for the report. Technically this is working as intended (the rule just checks the start indentation of block comments, since we can't be sure about the intended indentation of the following lines). I think it might be feasible to fix this within the `indent` rule, but there would be a few edge cases to deal with:

```js
  /* the first line of this comment
should not be indented */
```

The first line has an indentation of 2, but it shouldn't be indented at all. However, we can't move the second line to the left by 2, because there's not enough room. So we wouldn't be able to preserve the line alignment.

When the [multiline-comment-style](https://github.com/eslint/eslint/issues/8320) rule is released, it will be able to fix this case too.

**Comment by epmatsw** [comment]
Wow, yeah, multiline-comment-style would be perfect for this.

**Comment by not-an-aardvark** [comment]
Closing because this is solved by multiline-comment-style.
